### PR TITLE
refactor: removed comments from fs_calls.rs

### PR DESCRIPTION
**Pull request by ChinmayShringi**
## Description

Fixes # (issue)

 

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

- Test A - `lind_project/tests/test_cases/test_a.c`
- Test B - `lind_project/tests/test_cases/test_b.c`

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to a pull request and/or merged in other modules (native-client, lind-glibc, lind-project)

### CODE REVIEW

File: src/safeposix/syscalls/fs_calls.rs [hunk 1]
```diff
@@ -28,10 +26,7 @@ use std::ffi::CString;
 use std::ptr;
 use std::mem;
 
-// use crate::example_grates::vanillaglobal::*;
 use crate::example_grates::dashmapvecglobal::*;
-// use crate::example_grates::muthashmaxglobal::*;
-// use crate::example_grates::dashmaparrayglobal::*;
```
Comment (Yaxuan-w) [inline src/safeposix/syscalls/fs_calls.rs]: Those are required to use different version of fdtables

File: src/safeposix/syscalls/fs_calls.rs [hunk 2]
```diff
@@ -54,8 +49,6 @@ impl Cage {
     */
     pub fn open_syscall(&self, path: &str, oflag: i32, mode: u32) -> i32 {
 
-        // Convert data type from &str into *const i8
```
Comment (Yaxuan-w) [inline src/safeposix/syscalls/fs_calls.rs:57]: should leave for explanation

**Review by Yaxuan-w — CHANGES_REQUESTED** [review]
Great job! Minor changes needed. We'll merge this after we finish testing to ensure this pr won't break anything.

File: src/safeposix/syscalls/fs_calls.rs (hunk 2 above)
Comment (ChinmayShringi) [reply src/safeposix/syscalls/fs_calls.rs:57]: Reverted the deletion

File: src/safeposix/syscalls/fs_calls.rs (hunk 1 above)
Comment (ChinmayShringi) [reply src/safeposix/syscalls/fs_calls.rs]: Reverted changes

**Comment by Yaxuan-w** [comment]
I checked this PR by myself in order to do other porting work. We'll talk about testing for the following PRs. Thanks!

**Pull request merged**